npm_ls.txt

```diff
@@ -1,789 +1,1014 @@
-repoman@1.5.4 /Users/bastian/projekte/repoman
+repoman@1.6.0
```

Comment:
I don't really remember why I added this file in 2018. It doesn't seem to serve any purpose. I think we should remove it.

Reply:
I suspected you were using this track down deprecated dependencies (I used that output as well), but agree it doesn't need to be checked in.

Reply:
Yes, that sounds likely. But it can always be generated on demand, there is little value in having the file in the repo.